synapse/cortex.py

```diff
@@ -1719,6 +1820,24 @@ async def _initCoreQueues(self):
 
         self.multiqueue = await slab.getMultiQueue('cortex:queue', nexsroot=self.nexsroot)
 
+    async def _initStormGraphs(self):
+        path = os.path.join(self.dirn, 'slabs', 'graphs.lmdb')
+
+        slab = await s_lmdbslab.Slab.anit(path)
+        self.onfini(slab.fini)
+
+        self.graphs = s_lmdbslab.SlabDict(slab, db=slab.initdb('graphs'))
+
+        self.pkggraphs = {}
+        self.usergraphs = collections.defaultdict(dict)
+        self.globalgraphs = {}
+
+        for iden, gdef in self.graphs.items():
+            if gdef['scope'] == 'global':
+                self.globalgraphs[iden] = gdef
+            else:
+                self.usergraphs[gdef['creatoriden']] = gdef
```

Comment:
Is this missing an extra assignment? Seems like multiple graphs created by the same user would squish eachother?